src/MSBuildLocator/DotNetSdkLocationHelper.cs

```diff
@@ -15,11 +15,10 @@ internal static class DotNetSdkLocationHelper
     {
         private static readonly Regex DotNetBasePathRegex = new Regex("Base Path:(.*)$", RegexOptions.Multiline);
         private static readonly Regex VersionRegex = new Regex(@"^(\d+)\.(\d+)\.(\d+)", RegexOptions.Multiline);
+        private static readonly Regex SdkRegex = new Regex(@"(\d+\.\d+\.\d+) \[(.*)]$", RegexOptions.Multiline);
 
-        public static VisualStudioInstance GetInstance(string workingDirectory)
-        {
-            string dotNetSdkPath = GetDotNetBasePath(workingDirectory);
-
+        public static VisualStudioInstance GetInstance(string workingDirectory, string dotNetSdkPath)
```

Comment:
`workingDirectory` is now unused and can be removed.

Reply:
It has been removed.